src/main/java/duke/command/TaskManager.java

```diff
@@ -0,0 +1,248 @@
+package duke.command;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class TaskManager {
+
+    public static final String STATUS_DONE = "X";
+    public static boolean hasInvalidIndex = false;
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
```

Comment:
Since the methods are already non-static, consider removing static here as multiple initiation (then calling of TaskManager methods) will affect the `tasks` variable and subsequently your app.